src/client/components/forms/publication.js

```diff
@@ -16,7 +16,7 @@
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
-
+/* eslint valid-jsdoc: "warn" */
```

Comment:
Hopefully no need for this if we set "requireReturn" to false (http://eslint.org/docs/rules/valid-jsdoc)